src/index.css

```diff
@@ -27,3 +29,56 @@ code {
 .small-text {
   font-size: 10px;
 }
+
+.checkbox-container {
+  display: inline-flex;
+}
+
+.checkbox-container span {
+  padding-left: 0.5rem;
+}
+
+.regular-checkbox {
+  -webkit-appearance: none;
+  background: rgba(31, 89, 187, 0.05) !important;
+  border: 1px solid rgba(31, 89, 187, 0.5) !important;
+  box-sizing: border-box;
+  border-radius: 1px !important;
+  box-sizing: border-box;
+  display: inline-block;
+  margin: 0 !important;
+  /* padding: 0 !important; */
+  width: 13px !important;
+  height: 13px;
+
+  position: relative;
+}
+
+.regular-checkbox:active,
+.regular-checkbox:checked:active {
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05),
+    inset 0px 1px 3px rgba(0, 0, 0, 0.1);
+}
+
+.regular-checkbox:checked {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".regular-checkbox:checked" to come before selector ".regular-checkbox:checked:active" (no-descending-specificity)](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5691539)